sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/intellij/IntelliJCryptoUtil.java

```diff
@@ -0,0 +1,97 @@
+package com.azure.identity.implementation.intellij;
```

Comment:
it seems all files under `intellij` package is originally missing a license header